### DISCUSSION TITLE
collapsible in table

### DISCUSSION THREAD

**Discussion by liza-nt** [filamentphp/filament#8421] [Help]
Can we add a collapsible table and i do not need the collapsible layout of the table I need to collapse the whole table ..i have three table widgets i need to collapse the table

**Comment by untillnesss** [comment]
Wrap the table with Section.
[https://filamentphp.com/docs/3.x/support/blade-components/section#making-a-section-collapsible](https://filamentphp.com/docs/3.x/support/blade-components/section#making-a-section-collapsible)

**Reply by liza-nt** [reply]
but I don't have the blade file of the custom widget do I have to create it manually

**Reply by untillnesss** [reply]
Yes, that's right, you have to create a custom page that implements HasTable

**Reply by liza-nt** [reply]
Why do i need create custom page!!!
or I need to create blade page of widgets

**Reply by micraux** [reply]
Have you tried to embed your table into a `<div>` and use Tailwind classes (or create a custom class in your css file)?
https://tailwindcss.com/docs/visibility

**Reply by leoblanski** [reply]
Any idea @liza-nt ?
